components/project/post/RolesPage.tsx

```diff
@@ -39,7 +39,7 @@ export default ({ project }): JSX.Element => {
         <Typography className={classes.title}>
           <span style={{ fontWeight: 'bold' }}>{`Roles`}</span>
         </Typography>
-        <ArrayForm label="Roles" items={roles} setItems={setRoles} />
+        <ArrayForm label="Roles" items={roles} setItems={setRoles} options={roles} />
```

Comment:
- need to add options prop in line 26, then pass that into ArrayForm's options
- modify pages/project/post to fetch from backend (I did teamSize and duration choices for example)